test/functional/wallet_taproot.py

```diff
@@ -441,11 +441,11 @@ def run_test(self):
 
         self.log.info("Sending everything back...")
 
-        txid = self.rpc_online.sendtoaddress(address=self.boring.getnewaddress(), amount=self.rpc_online.getbalance(), subtractfeefromamount=True)
+        txid = self.rpc_online.sendall(recipients=[self.boring.getnewaddress()])["txid"]
         self.generatetoaddress(self.nodes[0], 1, self.boring.getnewaddress(), sync_fun=self.no_op)
         assert(self.rpc_online.gettransaction(txid)["confirmations"] > 0)
 
-        psbt = self.psbt_online.walletcreatefundedpsbt([], [{self.boring.getnewaddress(): self.psbt_online.getbalance()}], None, {"subtractFeeFromOutputs": [0]})['psbt']
+        psbt = self.psbt_online.sendall(recipients=[self.boring.getnewaddress()], options={"psbt": True})["psbt"]
```

Comment:
I was today years old, when I realized that

```suggestion
        psbt = self.psbt_online.sendall(recipients=[self.boring.getnewaddress()])["psbt"]
```

would also work, because the wallet automatically returns a PSBT when it cannot sign for a transaction. I do prefer it explicitly stated as proposed by @ishaanam, though.